src/components/layout/footer.js

```diff
@@ -242,44 +243,83 @@ const mobile_accordion_header = {
     backgroundColor: 'var(--color-grey-25)',
     boxShadow: 'none',
 }
+
 const mobile_accordion_header_about = Object.assign({}, mobile_accordion_header)
+
 const SocialWrapperComponent = () => {
```

Comment:
`SocialWrapperComponent` and  `CareerSocialWrapperComponent` Share almost the same data. Can we combine them?